alpine/Dockerfile

```diff
@@ -16,7 +16,10 @@ LABEL vendor="Intelsdi-X" \
 
 EXPOSE 8181
 
-RUN apk add --update curl && \
+RUN apk update && \
+    apk add curl && \
+    apk add bash && \
+    apk add jq && \
```

Comment:
Are we using jq anywhere? I didn't see it in the test scripts. Wondering if we should keep this image as minimal as possible.